src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/BlazorDebug/BlazorDebugConfigurationProvider.ts

```diff
@@ -0,0 +1,94 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+// import { spawn } from 'child_process';
+import * as vscode from 'vscode';
+
+import { RazorLogger } from '../RazorLogger';
+import { onDidTerminateDebugSession } from './Events';
+
+export class BlazorDebugConfigurationProvider implements vscode.DebugConfigurationProvider {
+
+    constructor(private readonly logger: RazorLogger, private readonly vscodeType: typeof vscode) { }
+
+    public async resolveDebugConfiguration(folder: vscode.WorkspaceFolder | undefined, configuration: vscode.DebugConfiguration): Promise<vscode.DebugConfiguration | undefined> {
+        const shellPath = process.platform === 'win32' ? 'cmd.exe' : 'dotnet';
+        const shellArgs = process.platform === 'win32' ? ['/c', 'chcp 65001 >NUL & dotnet run'] : ['run'];
+        const spawnOptions = {
+            cwd: configuration.cwd || folder && folder.uri && folder.uri.fsPath,
+            env: {
+                ...process.env,
+                ASPNETCORE_ENVIRONMENT: 'Development',

```

Comment:
When launching the dev server (for "standalone" projects) the env var is irrelevant, since the dev server is hardcoded to be in Development environment (it's in the name!)

When launching "hosted" projects, the env var is used as normal for the ASP.NET Core server.

Reply:
I went ahead and remove this. It seems sensible to accept whatever is in launchSettings and not override it.

I also removed the `process.env` passing since it looks like VS Code does this in the terminal instance by default ([ref](https://github.com/microsoft/vscode/blob/62595dbd397f55e8caa41208d78a4569b3ec2d6f/src/vs/workbench/parts/terminal/electron-browser/terminalInstance.ts#L597))